docker/Dockerfile

```diff
@@ -74,9 +74,9 @@ ENV CPPFLAGS="-I/usr/include -Duse_LARGEFILE -DMAXFIELDMETHODS_=500 -DGFS_PHYS"
 ENV FCFLAGS="-fcray-pointer -Waliasing -ffree-line-length-none -fno-range-check -fdefault-real-8 -fdefault-double-8 -fopenmp"
 
 COPY FMS /FMS
-RUN cd /FMS && autoreconf --install && ./configure 
-RUN cd /FMS && make -j8
-RUN mv /FMS/*/*.mod /FMS/*/*.o /FMS/*/*.h FMS/
+RUN cd /FMS && autoreconf --install && ./configure && \
+    cd /FMS && make -j8 && \
+    mv /FMS/*/*.mod /FMS/*/*.o /FMS/*/*.h /FMS/
```

Comment:
No need to do an explicitly copy here.  Can add a target destination directory in the configure call and then call a make install after the make is finished.

Reply:
If you know how to do this and can suggest a code change (the little +/- icon inside some papers next to the Preview button in this reply text box), I'll put it in. I agree that would be much better, but I don't know how to do it myself so it would take a fair bit of time to figure out.